scripts/compile_DL_AdditionalMessage.sh

```diff
@@ -0,0 +1,9 @@
+g++ DL_AdditionalMessage.cpp -lzmq -o ./build/DL_AdditionalMessage.exe
```

Comment:
u don't need to create all these scripts by copy pasting , u can put them in one file or a one line command, and also u don't need the .exe file for this kinds of files/ or libraries which doesn't start a process of sending or receiving.

for now u don't need to build these files, please remove all the added scripts.

Reply:
I delete all